lib/rex/parser/arguments.rb

```diff
@@ -21,7 +21,24 @@ class Arguments
       #
       def initialize(fmt)
         self.fmt = fmt
-        self.longest = fmt.keys.max_by(&:length)
+        self.longest = ''
+        fmt.each_key do |key|
+          if key.is_a?(Array)
```

Comment:
I wonder if instead of special casing for the key being an array or a string you could coerce everything to be an array?